awx/main/utils/common.py

```diff
@@ -65,21 +65,6 @@ def get_object_or_400(klass, *args, **kwargs):
         raise ParseError(*e.args)
 
 
-def get_object_or_403(klass, *args, **kwargs):
```

Comment:
You need to remove it from the `__all__` as well.

Reply:
https://github.com/ansible/awx/pull/3234/files#diff-1b22494f51df57adb72248c06506a997R39

Reply:
:scissors: